fluent-bundle/src/memoizer.rs

```diff
@@ -0,0 +1,17 @@
+use crate::types::FluentType;
+use intl_memoizer::Memoizable;
+use unic_langid::LanguageIdentifier;
+
+pub trait MemoizerKind: 'static {
+    fn new(lang: LanguageIdentifier) -> Self
+    where
+        Self: Sized;
```

Comment:
Probably make the trait bound `Sized + 'static`, remove the two `Self: Sized` bounds?

Reply:
We can't do that, the third method is called on the trait object and cannot have this bound. The bound is necessary on the first and second one so that it's not included on the object and doesn't affect object safety.